### PR TITLE
Rank in ci

### PR DESCRIPTION
**Pull request by harrisonpim**
~closes~ for https://github.com/wellcomecollection/platform/issues/5350

### CODE REVIEW

File: .buildkite/scripts/test_rank.sh [hunk 1]
```diff
@@ -36,7 +36,22 @@ ROOT=$(git rev-parse --show-toplevel)
 cp "$ROOT/search/src/test/resources/WorksMultiMatcherQuery.json" "$ROOT/rank/public/WorksMultiMatcherQuery.json"
 cp "$ROOT/search/src/test/resources/ImagesMultiMatcherQuery.json" "$ROOT/rank/public/ImagesMultiMatcherQuery.json"
 
-INDEX=$(curl "https://api.wellcomecollection.org/catalogue/v2/_elasticConfig" | jq -r .worksIndex)
+# case statement to choose the correct index
```
Comment (alexwlchan) [inline .buildkite/scripts/test_rank.sh:39]: nit: bin this comment

**Review by alexwlchan — APPROVED** [review]
Let's leave the ticket open until we've seen it run in a deploy.

**Comment by alexwlchan** [comment]
```
$ jest --queryEnv=candidate --index=works-indexed-2021-11-29a --testId=alternative-spellings --testId=precision --testId=recall --testId=false-positives
--
  | PASS __tests__/eval.test.ts (5.76 s)
  | ✓ works-indexed-2021-11-29a candidate alternative-spellings (170 ms)
  | ✓ works-indexed-2021-11-29a candidate precision (26 ms)
  | ✓ works-indexed-2021-11-29a candidate recall (15 ms)
  | ✓ works-indexed-2021-11-29a candidate false-positives (24 ms)
  |  
  | Test Suites: 1 passed, 1 total
  | Tests:       4 passed, 4 total
  | Snapshots:   0 total
  | Time:        5.862 s
  | Ran all test suites.
  | Done in 6.66s.
  |  
```

Should we be running the image rank tests as well?

**Comment by harrisonpim** [comment]
>Should we be running the image rank tests as well?

We should! I've just created an issue for it:
- https://github.com/wellcomecollection/platform/issues/5383

**Pull request merged**